esp32-srne-asf-example.yaml

```diff
@@ -772,13 +772,23 @@ switch:
   - platform: modbus_controller
     use_write_multiple: true
     modbus_controller_id: inverter0
-    name: "Alarms"
+    name: "Buzzer"
     register_type: holding
     address: 0xE210
     bitmask: 1
     entity_category: config
     icon: "mdi:toggle-switch"
 
+  - platform: modbus_controller
+    use_write_multiple: true
+    modbus_controller_id: inverter0
+    name: "Source Change Alarm"
```

Comment:
Could you explain the new feature to me? What gets indicated here exactly?

Reply:
Sure. This enables/disables the buzzer alarm for when the main input source changes.

Example: when there's a grid outage or when it recovers.

Reply:
No. The previous setting is global and supersedes any other. It must be enabled for this alarm to sound. If disabled, this alarm will not sound even if enabled.

Reply:
`Source Change Alarm` is fine.